packages/@react-spectrum/picker/test/Picker.test.js

```diff
@@ -1944,7 +1944,8 @@ describe('Picker', function () {
     beforeEach(() => {
       focusSpies = {
         onFocus: jest.fn(),
-        onBlur: jest.fn()
+        onBlur: jest.fn(),
+        onFocusChange: jest.fn()
```

Comment:
Mind adding a test to check that onFocusChange is being called now that you've added it to the spies here?

Reply:
Hey Daniel, we already had a test below that check if the `focusSpies` are being called:
```
expect(focusSpies.onFocus).toHaveBeenCalled();
```

Reply:
It doesn't check that onFocusChange is being fired with the correct values when the Picker is focused or blurred

Reply:
Thanks! sorry i didn't notice that. added in the newest commits!